### ISSUE TITLE
Highlight generation mistakes

### ISSUE THREAD

**Issue by ccpalmer** [GitMurf/roam-highlighter#53]
Here's what I highlighted:

![Screen Shot 2020-07-24 at 2 56 06 PM](https://user-images.githubusercontent.com/4670554/88425724-e011d700-cdbd-11ea-9c98-854ce7ec907e.png)

and here's what was captured by the highlighter:

```
- <h2>1. Classical vs Quantum Bits ||h3||1.1 Statevectors In quantum physics we use __statevectors__ to describe the state of our system. Say we wanted to describe the position of a car along a track, this is a classical system so we could use a number 𝑥 x x :</h2>
```

Note the `||h3||` and the three 'x's near the end.

**Comment by GitMurf** [comment]
@ccpalmer can you provide the website link to get to this page?

**Comment by GitMurf** [comment]
@ccpalmer can you please provide a link? I would like to troubleshoot myself but need to be able to test the exact page. Thanks!

**Comment by GitMurf** [comment]
@ccpalmer I was able to find the website via googling some of the text. I believe it is this one? https://qiskit.org/textbook/ch-states/representing-qubit-states.html

For your first issue, I am not having those problems. It copies as `<h2>` and `<h3>` as expected. I tried it two different ways of highlighting it as well. See GIF.

![oa1qgrPtgA](https://user-images.githubusercontent.com/64155612/96834074-3a8af580-13f6-11eb-98c0-7d6d1eb5cbae.gif)

As for your second issue with the "3 x's"... that is actually how the website has it stored in the HTML. See the screenshot below. Unfortunately that will be pretty hard to work logic in to handle that. I have NEVER seen it "this bad" with regards to weird HTML so this is definitely a corner case unfortunately.

![image](https://user-images.githubusercontent.com/64155612/96833357-22ff3d00-13f5-11eb-9924-cf495a87e87a.png)
